specifications/api/swagger.yaml

```diff
@@ -1840,12 +1840,363 @@ x-responsesErr: &responsesErr
       description: A specific error code for this error, more precise than HTTP ones.
       example: an_error_code
 
+x-responsesErrWalletNotFound: &responsesErrWalletNotFound
+  type: object
+  required:
+    - message
+    - code
+  properties:
+    message:
+      type: string
+      description: A descriptive error message.
```

Comment:
Instead of giving a description of the what the error message look like, wouldn't it be better to give some details about the circumstances which may lead to such error? e.g.

```
description: | 
  May occur when a given walletId does not match with any known 
  wallets (because it has been deleted, or has never existed).
```